src/agents/read_agent.py

```
@@ -1,51 +1,143 @@
+
+        # Initialize tools
+        self.field_tool = FieldRetrievalTool()
+        self.tools = [self.field_tool]
+
+        # Create ReAct agent prompt using the hub template
+        from langchain import hub
+        
+        try:
+            # Use the standard ReAct prompt from LangChain Hub
+            self.prompt = hub.pull("hwchase17/react")
+        except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)